projects/plugins/crm/src/rest-api/exceptions/class-base-exception.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Base Exception.
+ *
+ * @package Automattic\Jetpack_CRM\REST_API
+ */
+
+namespace Automattic\Jetpack_CRM\REST_API\Exception;
+
+defined( 'ABSPATH' ) || exit;
+
+use Exception;
+use Throwable;
```

Comment:
Cool, I usually use the backslash `\Exception` to refer to global classes.

Reply:
I'm actually kinda mixed 😅 
I've started to generally add all "dependencies" at the top of the file so it fits the React import pattern, but I'd be up for whatever the team wants to do as our standard (WordPress core doesn't use namespaces, so they don't have anything we can lean up against)

Reply:
I don't mind. I'm easy here, without any strong opinion here. The `use` clause doesn't add relevant information to me as a developer, only increases the number of lines 😆. I usually use `use` for those long namespaces and use only the class name.

Reply:
In theory it should make the code more DRY since you're able to replace it if it's using the same interface. E.g.:

If we start out using the native `\Exception` class but chose to extend it with our own `class JPCRM_Exception extends \Exception {}` then we'd only have to replace the use statement at the top of the file instead of the places all uses in the code itself, so it becomes more of an actual "import" (like in JS) versus just being a quality of life improvement not having to write the full namespace every time we want to use classes 😛